packages/client/src/rpc/modules/debug.ts

```diff
@@ -92,6 +101,7 @@ export class Debug {
       [validators.transaction()],
       [validators.blockOption],
     ])
+    this.storageRangeAt = middleware(this.storageRangeAt.bind(this), 5, [[validators.hex]])
```

Comment:
the validators here will only validate the first arg, reading from comments need 5 validators